Modules/xASL_module_BIDS2Legacy.m

```diff
@@ -102,7 +102,26 @@
             %% 2. Parse modality
             
             pathLegacy_SubjectSession = fullfile(x.dir.xASLDerivatives, SubjectSession);
-            
+            bFolderExisted = exist(pathLegacy_SubjectSession, 'dir')==7;
+
+            % Backwards compatibility: rename sub-*** to sub-***_1
+            if strcmp(SessionID, '1')
+                pathLegacy_SubjectSession_OLD = fullfile(x.dir.xASLDerivatives, SubjectID);
+                if ~bFolderExisted && exist(pathLegacy_SubjectSession_OLD, 'dir')==7
+                    warning(['Renaming ' pathLegacy_SubjectSession_OLD '->' SubjectSession]);
+                    xASL_Move(pathLegacy_SubjectSession_OLD, pathLegacy_SubjectSession, 1, 0);
+                    bFolderExisted = true;
+                end
+            end
+
+            % Provide warning that we are going to overwrite the folder
+            if bFolderExisted
+                warning(['Overwriting rawdata copied to: ' pathLegacy_SubjectSession]);
+                fprintf('%s\n', 'Note that we do not overwrite any derivatives');
```

Comment:
You mean, we don't overwrite processed files. Cause the origin files derivatives/ExploreASL/sub1/ASL_1/ASL.nii is kind of a derivative file.

In other words, we are not overwriting derived files, but this message can be confusing and the user might think we talk about the entire "derivatives" folder.

Reply:
a750e39ae434b87ce24d127bfc2a24d87652ab95